### PR TITLE
fix broken links for KubeletConfiguration struct

### PR DESCRIPTION
**Pull request by alexandrevilain**
Hi!

For some configmap and secret documentation pages, links to the KubeletConfiguration struct are broken.
It's due to the usage of the `docsbranch` parameter which is evaluated to `main`. But the  `main` branch doesn't exist in the [kubernetes](https://github.com/kubernetes/kubernetes) repo. 

This PR replaces the parameter to an hardcoded value: `master`. 

You can check that the link is currently broken here: 
- https://kubernetes.io/fr/docs/concepts/configuration/secret/#les-secrets-mont%C3%A9s-sont-mis-%C3%A0-jour-automatiquement (latest version)
- https://v1-23.docs.kubernetes.io/fr/docs/concepts/configuration/secret/#les-secrets-mont%C3%A9s-sont-mis-%C3%A0-jour-automatiquement (v1.23 version)

### CODE REVIEW

**Comment by electrocucaracha** [comment]
Thanks for fixing this issue. Usually this kind of PRs are breakdown in multiple PRs, one per language.

/lgtm

**Comment by nasa9084** [comment]
/assign
/approve
for /ja page

**Review by raelga — APPROVED** [review]

**Comment by divya-mohan0209** [comment]
Hello @alexandrevilain , please limit your PRs to one language file ONLY, irrespective of how trivial the changes are. Please could you PR the changes in for the English localization only via this PR & get it lgtm'ed & approved? Once done, please consider making changes in other localization files.

**Comment by divya-mohan0209** [comment]
Hello @alexandrevilain , please can you limit the PRs to only one localization i.e. the main website in English and resubmit a commit?

**Comment by alexandrevilain** [comment]
Hello @divya-mohan0209, 
For sure I can do that :)

**Comment by alexandrevilain** [comment]
Done @divya-mohan0209 🙂
I close this PR.

**Pull request closed without merging**
